node/src/bootstrapper.rs

```diff
@@ -1865,6 +1899,7 @@ mod tests {
             &self,
             config: BootstrapperConfig,
             _actor_factory: Box<dyn ActorFactory>,
+            _persist_config: &dyn PersistentConfiguration,
```

Comment:
I'm a little concerned that ignoring this parameter may mean that its value isn't tested. Maybe there's an integration test for this somewhere.